src/AspNetCoreModuleV2/CommonLib/StringHelpers.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include <string>
+
+[[nodiscard]]
+bool ends_with(const std::wstring &source, const std::wstring &suffix, bool ignoreCase = false);
+
+template<typename ... Args>
+[[nodiscard]]
+std::wstring format(const std::wstring& format, Args ... args)
```

Comment:
nit: either move this to StringHelpers.cpp or consolidate into one file.

Reply:
`template<typename ... Args>` method can't be moved to cpp unfortunately.